packages/actor-query-operation-filter-sparqlee/test/ActorQueryOperationFilterSparqlee-test.ts

```diff
@@ -148,6 +149,22 @@ describe('ActorQueryOperationFilterSparqlee', () => {
       await new Promise<void>(resolve => output.bindingsStream.on('error', () => resolve()));
     });
 
+    it('Should log warning for an expressionError', async() => {
+      // eslint-disable-next-line @typescript-eslint/consistent-type-assertions
```

Comment:
`eslint-disable-next-line` should not be used unless absolutely necessary. I think they can be resolved here, so there is no need for them I think.